index.html

```diff
@@ -0,0 +1,499 @@
+          <button class="button choose-plan__button" type="button">
+            Select
+          </button>
+        </div>
+        <div class="choose-plan__item">
+          <img
+            src="./img/illustration-free.svg"
+            width="145"
+            height="165"
+            alt="illustration-free"
+          />
+          <p class="choose-plan__description">Free Plan</p>
+          <ul class="choose-plan__list--item">
+            <li class="choose-plan__list">
+              <img
+                src="/img/check-2.svg"
+                width="24"
+                height="24"
+                alt="check-2"
+              />
+              <p class="choose-plan__text">Unlimited Bandwitch</p>
+            </li>
+            <li class="choose-plan__list">
+              <img
+                src="/img/check-2.svg"
+                width="24"
+                height="24"
+                alt="check-2"
+              />
+              <p class="choose-plan__text">Encrypted Connection</p>
+            </li>
+            <li class="choose-plan__list">
+              <img
+                src="/img/check-2.svg"
+                width="24"
+                height="24"
+                alt="check-2"
+              />
+              <p class="choose-plan__text">No Traffic Logs</p>
+            </li>
+            <li class="choose-plan__list">
+              <img
+                src="/img/check-2.svg"
+                width="24"
+                height="24"
+                alt="check-2"
+              />
+              <p class="choose-plan__text">Works on All Devices</p>
+            </li>
+          </ul>
+          <h3 class="choose-plan__subtitle">Free</h3>
+          <button class="button choose-plan__button" type="button">
+            Select
+          </button>
+        </div>
+      </section>
+
+      <section class="global-net">
+        <h2 class="global-net__title">Huge Global Network of Fast VPN</h2>
+        <p class="global-net__subtitle">
+          See <strong class="text-accent">LaslesVPN</strong> everywhere to make
+          it easier for you when you move locations.
+        </p>
+        <img
+          src="./img/huge-global.svg"
+          width="1060"
+          height="538"
+          alt="global-lassmap"
+        />
+        <img
+          src="./img/sponsored.svg"
+          width="1136"
+          height="538"
+          alt="logo-sponsores"
+        />
+      </section>
+
+      <section class="trusted-customer">
+        <h2 class="title trusted-customer__title">
+          Trusted by Thousands of Happy Customer
+        </h2>
+        <p class="trusted-customer__subtitle">
+          These are the stories of our customers who have joined us with great
+          pleasure when using this crazy feature.
+        </p>
+        <div class="trusted-customer__slider">
+          <div class="trusted-customer__item">
+            <div class="trusted-customer--datauser">
```

Comment:
Here and further:
I assume, you wanted to use `__` instead of `--` :)

Reply:
+